testing/matrix-sdk-integration-testing/src/tests/sliding_sync/room.rs

```diff
@@ -538,3 +550,299 @@ async fn test_room_notification_count() -> Result<()> {
 
     Ok(())
 }
+
+static DROP_TODEVICE: StdMutex<bool> = StdMutex::new(true);
+fn drop_todevice(response: &mut Bytes) {
```

Comment:
Let the code breathe :)

```suggestion
static DROP_TODEVICE: StdMutex<bool> = StdMutex::new(true);

fn drop_todevice(response: &mut Bytes) {
```